src/server/Document/Renderers/CanonicalRenderer.tsx

```diff
@@ -35,6 +35,25 @@ export default function CanonicalRenderer({
   return (
     <html lang="en-GB" className={NO_JS_CLASSNAME} {...htmlAttrs}>
       <head>
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `
+            window.__reverb = {};
+            window.__reverb.__reverbLoadedPromise = new Promise((resolve, reject) => {
+              window.__reverb.__resolveReverbLoaded = resolve;
+              window.__reverb.__rejectReverbLoaded = reject;
+            });
+
+            window.__reverb.__reverbTimeout = setTimeout(() => {
+              window.__reverb.__rejectReverbLoaded();
+            }, 5000);`,
+          }}
+        />
+        <script
+          async
+          src="https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/reverb-3.9.2.js"
+        />
```

Comment:
Doesn't need to be now, but may be worth extracting this into its own component so that we can re-use it in the Next.js app as well.

Reply:
I was looking at the [Next.js boilerplate](https://github.com/bbc/simorgh/blob/latest/ws-nextjs-app/pages/_document.page.tsx) to establish how to pass in the Reverb script.
A separate template as recommended, addresses this.
I'll raise a ticket to implement this after enabling Reverb on TEST.

Reply:
I've raised WSTEAMA-1552 to address this.